src/index.html

```diff
@@ -1,14 +1,264 @@
+              <li class="service__item">Сontent Production</li>
+              <li class="service__item">Graphic Design</li>
+              <li class="service__item">Video Production</li>
+              <li class="service__item">Post Production</li>
+            </ul>
+            <button class="service__button service__button--blue"></button>
+          </div>
+          <div class="service">
+            <span class="service__number service__number--green">003</span>
+            <ul class="service__list">
+              <li class="service__item">Marketing Strategy</li>
+              <li class="service__item">Email Marketing</li>
+              <li class="service__item">Paid Advertising</li>
+              <li class="service__item">Blog Content & SEO</li>
+            </ul>
+            <button class="service__button service__button--green"></button>
+          </div>
+          <div class="service">
+            <span class="service__number service__number--red">004</span>
+            <ul class="service__list">
+              <li class="service__item">Digital Communications</li>
+              <li class="service__item">Influencer Marketing</li>
+              <li class="service__item">Product Placements</li>
+              <li class="service__item">Strategic Partnerships</li>
+            </ul>
+            <button class="service__button service__button--red"></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="approach">
+      <div class="approach__wrapper">
+        <div class="approach__block-text">
+          <span class="approach__decor-text">approach</span>
+          <h2 class="approach__subtitle">We strive to create meaningful projects</h2>
+          <p class="approach__paragraph approach__paragraph--1">
+            We are sure that the brands built on a strong strategy, big ideas and simple human values always win. Only such brands change the game.
+          </p>
+          <p class="approach__paragraph approach__paragraph--2">
+            No one can realize the idea as its creators. Therefore, we set up agency-based photo and video production departments.
+          </p>
+          <p class="approach__paragraph approach__paragraph--3">
+            We strive to create meaningful projects that make brands valuable, and the world around us more aesthetic.
+          </p>
+        </div>
+      </div>
+      <div class="approach__image"></div>
+    </section>
+    <section class="testimonials">
+      <div class="testimonials__wrapper">
+        <div class="testimonials__block-text">
+          <span class="testimonials__decor-text">testimonials</span>
+          <h2 class="testimonials__subtitle">What people say</h2>
+        </div>
+        <div class="testimonials__people">
+          <div class="person">
+            <div class="person__wrap-img person__wrap-img--1">
+              <img src="images/testimonials/person-1.png" alt="image Tal Gilard" class="person__img">
```

Comment:
```suggestion
              <img src="images/testimonials/person-1.png" alt="Tal Gilard" class="person__img">
```
скринридер и так озвучивает, что это картинка